src/openforms/config/templatetags/privacy_policy.py

```diff
@@ -0,0 +1,18 @@
+from django import template
+from django.template import Context, Template
+
+from ..models import GlobalConfiguration
+
+register = template.Library()
+
+
+@register.simple_tag()
+def privacy_policy():
+    conf = GlobalConfiguration.get_solo()
+    if conf.privacy_policy_url:
+        template_string = '{% load i18n %}<a href="{{ privacy_policy }}">{% trans "privacy policy" %}</a>'
+        return Template(template_string).render(
+            Context({"privacy_policy": conf.privacy_policy_url})
+        )
+
+    return ""
```

Comment:
Please implement this as an [`inclusion_tag`](https://docs.djangoproject.com/en/2.2/howto/custom-template-tags/#inclusion-tags) instead.

Reply:
I did this initially, but I couldn't figure out how to deal with the new lines that get added. So I went for a simple tag instead.

Reply:
newlines should be ignored by the HTML related assertions such as https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertHTMLEqual

Reply:
But even if the new lines are ignored by tests, it still looks funny to have a checkbox like this:
```
I have read the 
privacy_policy
and agree with it
```
instead of having it all in one line?

Reply:
hrm it is odd that those are converted into linebreaks :thinking: let's get this merged as is then, and possibly improve it later